src/structurizr/view/view.py

```diff
@@ -125,6 +125,11 @@ def model(self) -> Model:
         """Return the `Model` for this view."""
         return self.software_system.get_model()
 
+    @property
+    def relationship_views(self) -> Iterable[RelationshipView]:
+        """Return the relationship views contained by this view."""
+        return self._relationship_views
```

Comment:
Should this return an iterator over the views or a copy of the set?

Reply:
I've changed to construct a new set - this might confuse folk slightly as in DynamicView it'll be a list, but we're clear here that they should only expect it to be Iterable so I think that's OK.

Reply:
Sounds good to me.